ksqldb-serde/src/main/java/io/confluent/ksql/serde/avro/AvroDataTranslator.java

```diff
@@ -117,7 +119,9 @@ private static Object replaceSchema(final Schema schema, final Object object) {
 
       case STRUCT:
         return convertStruct((Struct) object, schema);
-
+      case BYTES:
+        return DecimalUtil.ensureFit(
```

Comment:
Why don't we just use `public static BigDecimal ensureFit(final BigDecimal value, final Schema schema)`?